src/sentry/static/sentry/app/components/smartSearchBar/index.tsx

```diff
@@ -705,21 +705,29 @@ class SmartSearchBar extends React.Component<Props, State> {
    * Fetches latest releases from a organization/project. Returns an empty array
    * if an error is encountered.
    */
-  fetchReleases = async (query: string): Promise<any[]> => {
+  fetchReleases = async (releaseHash: string): Promise<any[]> => {
     const {api, organization} = this.props;
     const {location} = this.context.router;
 
     const project = location && location.query ? location.query.projectId : undefined;
 
+    const url = `/organizations/${organization.slug}/releases/`;
+    const fetchQuery: {[key: string]: string | number} = {
```

Comment:
Could the `Query` type from location package help here?

Reply:
Unfortunately not, the `Query` type enforces values to be strings. We want to allow numbers as values here.